src/Compilers/VisualBasic/Portable/VBResources.resx

```diff
@@ -5476,4 +5476,10 @@
   <data name="ERR_BadSourceCodeKind" xml:space="preserve">
     <value>Provided source code kind is unsupported or invalid: '{0}'</value>
   </data>
+  <data name="ERRWRN_Last_Title" xml:space="preserve">
+    <value>Unable to load analyser assembly</value>
```

Comment:
This one needs to be checked as I had add it, so I could compile and test. So could be an issue when merging.